### ISSUE TITLE
Stable files in production

### ISSUE THREAD

**Issue by ddamato** [KiwiKilian/eleventy-plugin-og-image#157]
I'd like to see the option of stable file names for production. I'm interested in predetermining the URL to the image on another one of my sites and having the filename hash makes this difficult. Looks like it might be as simple as [exposing the `previewFilenames` Boolean](https://github.com/KiwiKilian/eleventy-plugin-og-image/blob/main/.eleventy.js#L71) in the plugin options (maybe as a new name: `stableFilenames`?).

**Comment by KiwiKilian** [comment]
I guess a custom generator function like `generateOutputFileName` with a couple of parameters (like page, svg, data) would also do the trick right?

**Comment by ddamato** [comment]
Sounds even better!

**Comment by KiwiKilian** [comment]
Happy to hear if `getOutputFileSlug` serves your use case!

**Comment by ddamato** [comment]
Looks good! A few things.

- I was running `@11ty/eleventy@1.1.1` and I was getting a `File is not a function` error at `renderOgImage`. When I upgraded to `@11ty/eleventy@2.0.1` no more error with the same settings until...
- I tried using `npx @11ty/eleventy --serve` for the first time (I used to just serve the static files, that doesn't seem to work now) and all of the posts that had a `draft: true` set would fail during the `slug` because the value was a Boolean. Luckily using the new `getOutputFileSlug` function solved this by doing `getOutputFileSlug: ({ context }) => context.page.fileSlug,` (which is exactly my original ask).
